HISTORY.rst

```diff
@@ -1,15 +1,24 @@
 =======
 History
 =======
-3.58.0 (2022-04-16)
+3.59.0 (2022-05-26)
+-------------------
+
+Changes:
+
+- Setting ``repo_url`` in your project config is no longer required to run ``metadeploy_publish``. (#3200)
+- Adds new commands ``cci robot install_playwright`` and ``cci robot uninstall_playwright`` for installing and uninstalling experimental support for the robot framework Browser library which is based on Playwright. (#3194)
```

Comment:
```suggestion
- Adds new commands ``cci robot install_playwright`` and ``cci robot uninstall_playwright`` for installing and uninstalling experimental support for the Robot Framework ``Browser`` library, which is based on Playwright. (#3194)
```

Reply:
We might want to add "For more information see the section "Playwright Technology Preview" in the public documentation.